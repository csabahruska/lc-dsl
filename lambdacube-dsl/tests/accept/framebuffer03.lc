main =
    let frame = FrameBuffer (colorImage1 (V4F 0.0 0.0 0.4 1.0)
                            ,colorImage1 (V4F 1.0 1.0 1.0 1.0)
                            ,DepthImage #1 1.0
                            )
    in ScreenOut frame
