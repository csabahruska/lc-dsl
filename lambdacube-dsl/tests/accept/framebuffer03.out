Pipeline {textures = [], samplers = [], targets = [RenderTarget {renderTargets = [(Color,Just (Framebuffer Color)),(Color,Just (Framebuffer Color)),(Depth,Just (Framebuffer Depth))]}], programs = [], slots = [], commands = [SetRenderTarget 0,ClearRenderTarget [(Color,VV4F (V4 0.0 0.0 0.4 1.0)),(Color,VV4F (V4 1.0 1.0 1.0 1.0)),(Depth,VFloat 1.0)]]}